data-intake/ddi/README.md

```diff
@@ -0,0 +1,16 @@
+# Direct Data Intake API
+The Direct Data Intake API is suitable for Visier customers who have, at least in part, data to load into Visier that require no additional transformations. This is what makes this process 'direct'. In fact, the process is so streamlined that the column names in the source data files have match the names of the properties to load in the target objects.
+
+## Load Modes
+When provisioning data to Visier using the Data Direct API, it's important to understand the three available modes that correespond to the relationship Direct Data loading has to other means of provisioning data in you tenant.
+
+The following diagram outlines these modes:
```

Comment:
Do you think it'd make sense to add examples of when you'd use Supplemental mode over Primary mode? Regardless the graph helps a lot in parsing what the direct data intake API does so I don't think it's necessary but it might be nice for people who might be newer to Visier APIs.

Reply:
Yeah, I think that can help reinforce that understanding. I'll add some example scenarios.